packages/jellyfish-api-core/__tests__/category/loan/tokenSplits.test.ts

```diff
@@ -0,0 +1,966 @@
+  await testing.generate(1)
+
+  // set the dfip2203/active to true
+  await testing.rpc.masternode.setGov({ [attributeKey]: { 'v0/params/dfip2203/active': 'true' } })
+  await testing.generate(1)
+
+  // Retrieve and verify gov vars
+  const attributes = await testing.rpc.masternode.getGov(attributeKey)
+  expect(attributes.ATTRIBUTES['v0/params/dfip2203/active']).toStrictEqual('true')
+  expect(attributes.ATTRIBUTES['v0/params/dfip2203/reward_pct']).toStrictEqual(`${futRewardPercentage}`)
+  expect(attributes.ATTRIBUTES['v0/params/dfip2203/block_period']).toStrictEqual(`${futInterval}`)
+}
+
+async function checkTokenSplit (tokenID: string, tokenSymbol: string, tokenSuffix: string, minted: BigNumber, checkLoan: boolean, checkCollateral: boolean): Promise<string> {
+  // check old token
+  const oldToken = (await testing.rpc.token.getToken(tokenID))[tokenID]
+  expect(oldToken.symbol).toStrictEqual(`${tokenSymbol}${tokenSuffix}`)
+  expect(oldToken.minted).toStrictEqual(new BigNumber(0))
+  expect(oldToken.mintable).toStrictEqual(false)
+  expect(oldToken.tradeable).toStrictEqual(false)
+  expect(oldToken.finalized).toStrictEqual(true)
+  expect(oldToken.destructionTx).toStrictEqual(await testing.rpc.blockchain.getBestBlockHash()) // NOTE(surangap) - should it be the tx hash instead of bockhash
+  expect(oldToken.destructionHeight).toStrictEqual(new BigNumber(await testing.rpc.blockchain.getBlockCount()))
+
+  // old token in attributes
+  const attributes = await testing.rpc.masternode.getGov('ATTRIBUTES')
+  expect(attributes.ATTRIBUTES[`v0/token/${tokenID}/fixed_interval_price_id`]).not.toBeDefined()
+  if (checkCollateral) {
+    expect(attributes.ATTRIBUTES[`v0/token/${tokenID}/loan_collateral_enabled`]).not.toBeDefined()
+    expect(attributes.ATTRIBUTES[`v0/token/${tokenID}/loan_collateral_factor`]).not.toBeDefined()
+  }
+  if (checkLoan) {
+    expect(attributes.ATTRIBUTES[`v0/token/${tokenID}/loan_minting_enabled`]).not.toBeDefined()
+    expect(attributes.ATTRIBUTES[`v0/token/${tokenID}/loan_minting_interest`]).not.toBeDefined()
+  }
+  expect(attributes.ATTRIBUTES[`v0/locks/token/${tokenID}`]).not.toBeDefined()
+
+  // const tokenIDV1 = tokenID
+  tokenID = Object.keys(await testing.rpc.token.getToken(tokenSymbol))[0]
+  // check new token in attributes
+  expect(attributes.ATTRIBUTES[`v0/token/${tokenID}/fixed_interval_price_id`]).toStrictEqual(`${tokenSymbol}/USD`)
+  if (checkCollateral) {
+    expect(attributes.ATTRIBUTES[`v0/token/${tokenID}/loan_collateral_enabled`]).toStrictEqual('true')
+    expect(attributes.ATTRIBUTES[`v0/token/${tokenID}/loan_collateral_factor`]).toStrictEqual('1')
+  }
+  if (checkLoan) {
+    expect(attributes.ATTRIBUTES[`v0/token/${tokenID}/loan_minting_enabled`]).toStrictEqual('true')
+    expect(attributes.ATTRIBUTES[`v0/token/${tokenID}/loan_minting_interest`]).toStrictEqual('0')
+  }
+  expect(attributes.ATTRIBUTES[`v0/locks/token/${tokenID}`]).toStrictEqual('true')
```

Comment:
that's one more attribute
```
'v0/token/{id}/ascendant': '{num}/split'
```
what does this for huh??

Reply:
`ascendant` and `decendant` keeps the link between the token versions. set by defid when split happens